scout/parse/variant/clnsig.py

```diff
@@ -122,7 +125,7 @@ def is_pathogenic(variant):
         if clnsig in load_categories:
             return True
         if isinstance(clnsig, int):
-            if clnsig == 4 or clnsig == 5:
+            if clnsig == 4 or clnsig == 5 or clnsig == 8:
                 return True
```

Comment:
```
if clnsig in load_categories or clnsig in [4, 5, 8]:
            return True
```

Reply:
Right, that will actually work. I'll put them all in the same set!

Reply:
Loose typing is nice! 😁